.gitignore

```diff
@@ -10,6 +10,8 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+_tmp/
```

Comment:
_tmp is not used for now?

Reply:
`_tmp` is used in `hack/verify-xxx`, I add it handy.